renovate.json

```
@@ -29,5 +29,15 @@
       "matchPaths": ["**/__fixtures__/**"],
       "enabled": false
     }
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": ["config/options/index.ts"],
+      "matchStrings": [
+        "default: 'ghcr.io/containerbase/sidecar:(?<currentValue>.*)'"
+      ],
+      "depNameTemplate": "containerbase/sidecar",
```

```suggestion
        "default: '(?<depName>ghcr.io/containerbase/sidecar):(?<currentValue>.*)'"
      ]
```
published times are different, so we should same image as renovate source. also made it simpler.